components/system-broker/internal/director/pager.go

```diff
@@ -0,0 +1,98 @@
+package director
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+
+	gcli "github.com/machinebox/graphql"
+	"github.com/pkg/errors"
+)
+
+type GenericPage struct {
+	Data       interface{}       `json:"data"`
+	PageInfo   *graphql.PageInfo `json:"pageInfo"`
+	TotalCount int               `json:"totalCount"`
+}
+
+type GenericOutput struct {
+	Result *GenericPage `json:"result"`
+}
+
+type Pager struct {
```

Comment:
The idea is that this pager is generic by itself and should be put in the director component. At the moment I don't see the benefit to put in the director, but at the moment it will bring more complexity in my opinion